app/controllers/ctc/questions/no_dependents_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Ctc
+  module Questions
+    class NoDependentsController < QuestionsController
+      include AuthenticatedCtcClientConcern
+
+      layout "intake"
+
+      def self.show?(intake)
+        intake.dependents.none?
```

Comment:
the ticket doesnt explicitly indicate, but I wonder whether people with no QUALIFYING dependents should also see this flow... to do that, we'd need to run each dependent through the qualifications service.

I'm not super up to date with the context of the original ticket, so not sure whether it should be raw dependents being 0, or no dependents who qualify for benefits. 🤔

Reply:
yeah, the figma indicates some text about "eligible" dependents, so I guess there is a case where someone could have added 5 dependents, only one was originally qualifying, so we'd want to show this page if they remove the one qualifying dependent.

Reply:
Yeah @embarnard and I talked about it this afternoon and I'm going to confirm with Yvonne but it seems likely that it should only be if there are eligible dependents.